build/Rulesets/Roslyn_BuildRules.ruleset

```diff
@@ -81,23 +81,23 @@
     <Rule Id="RS0027" Action="Error" />
   </Rules>
   <Rules AnalyzerId="System.Collections.Immutable.Analyzers" RuleNamespace="System.Collections.Immutable.Analyzers">
-    <Rule Id="RS0012" Action="Warning" />
```

Comment:
Change in this file is required to make build success (those analyzers use IOperation, whereas this PR changes the interface)